src/utils/build_importer.py

```diff
@@ -0,0 +1,116 @@
+            item_type = (
+                driver.find_element(By.CLASS_NAME, "d4t-ItemLibrary")
+                .find_element(By.CLASS_NAME, "d4t-active")
+                .find_element(By.XPATH, "./../..")
+                .find_element(By.CLASS_NAME, "d4t-category-header")
+                .text
+            )
+        item_properties = driver.find_elements(By.CLASS_NAME, "d4t-item-mods")
+        try:
+            item_options = driver.find_element(By.CLASS_NAME, "d4t-item-options")
+        except NoSuchElementException:
+            continue
+        # skip uniques for now, need to figure out a better way to grab these
+        if item_options.text.startswith("Unique"):
+            driver.find_element(By.CLASS_NAME, "d4t-header-title").text
+            Logger.info(f"Skipping {driver.find_element(By.CLASS_NAME, "d4t-header-title").text} unique importing currently unsupported")
+            continue
+        item_details = {}
+        item_details["implicits"] = ""
+        if item_type != "Helm" and item_type != "Chest Armor" and item_type != "Gloves":
+            item_details["implicits"] = translate_modifiers(item_properties[0]) if len(item_properties) > 0 else ""
+            item_properties = item_properties[1:]
+        item_details["mods"] = translate_modifiers(item_properties[0]) if len(item_properties) > 0 else ""
+        if item_details["mods"] == "":
+            # normally means we've clicked the 2nd weapon slot when we've got a two-handed sword
+            continue
+        # commenting out for now, but leaving here to sort in the future
+        # item_details["tempers"] = item_properties[1] if len(item_properties) > 1 else ""
+        # item_details["aspect"] = item_properties[2] if len(item_properties) > 2 else ""
+        # item_details["gems"] = item_properties[3] if len(item_properties) > 3 else ""
+        item_dict[item_type] = item_details
+
+    driver.quit()
+    filter = convert_to_filter(item_dict)
+    with open(f"{Path(f"{os.path.expanduser("~")}/.d4lf")}/profiles/{filter_label}.yaml", "w") as file:
+        file.write(dump(filter, default_flow_style=None, sort_keys=False))
+
+    print(f"Created profile {Path(f"{os.path.expanduser("~")}/.d4lf")}\\profiles\\{filter_label}.yaml")
+
+
+def translate_modifiers(mods):
+    translated_mods = []
+    mods = mods.find_elements(By.CLASS_NAME, "d4t-property")
+    for mod in mods:
+        translated_mods += [[remove_extra_underscores(re.sub(r"[0-9]|,|\.|\+|\[|\]|-|%", "", mod.text).lower().strip().replace(" ", "_"))]]
+
+    return translated_mods
+
+
+def remove_extra_underscores(str):
+    i = 1
+    while i < len(str):
+        if str[i] == str[i - 1] and str[i] == "_":
+            str = str[:i] + str[(i + 1) :]
+            i -= 1
+        i += 1
+
+    return str
+
+
+def convert_to_filter(items):
```

Comment:
Instead of a dict, please use our (validated) classes. Makes it easier to maintain and adapt to future changes. Talking about `config.models.ProfileModel`. You can use the package `pydantic-yaml` to generate the yaml based on that object